### ISSUE TITLE
AddonManager proxy

### ISSUE THREAD

**Issue by Alexey-T** [Alexey-T/SynWrite#20]
@Andreas123456 Does this ver: proxy works? 
(copy files by hand)

[syn_plugin_manager.zip](https://github.com/Alexey-T/SynWrite/files/392205/syn_plugin_manager.zip)

**Comment by Andreas123456** [comment]
It does work.

**Comment by Alexey-T** [comment]
Very good

**Comment by Andreas123456** [comment]
FYI: After trying SynWrite's new proxy handing last weekend, I could not connect to my work PC from home office today (Monday). The PC was not reachable via ping. I asked a colleague in the office to reboot my PC, but it didn't help. Arggh!!! I had to give my password to him so that he could log in locally to my PC. After trying different things (playing with the network adapter settings etc.) for half an hour or so, he managed to re-enable IP connectivity of my PC, so that I can now reach it again from home office. I cannot say for sure whether the problem was caused by trying SynWrite's new proxy handing or whether it was just bad luck. I will never try it again.
